helloworld-jest-test-sample/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Tslint is deprecated, can this be changed to eslint?
Or just remove it entirely

Reply:
You can just copy this https://github.com/microsoft/vscode-extension-samples/blob/master/webpack-sample/.eslintrc.js